web_responsive/readme/DESCRIPTION.rst

```diff
@@ -103,16 +60,6 @@ This module adds responsiveness to web backend.
 
   .. image:: ../static/img/formview.gif
 
-* Set chatter on the side of the screen, optional per user
-
-  .. image:: ../static/img/chatter_sided.gif
-
 * Sticky chatter topbar
 
   .. image:: ../static/img/chatter_topbar.gif
-
-* When the chatter is configured on the side part, the document viewer fills that
-  part for side-by-side reading instead of full screen. You can still put it on full
-  width preview clicking on the new maximize button.
-
-  .. image:: ../static/img/document_viewer.gif
```

Comment:
Why has this been deleted?

Reply:
As its not needed in v16 because of the changes made in the default layout for attachment viewer

Reply:
What this module added was that the viewer opened only above the chatter without occupying the form. I don't see that this can be done in 16.0... but maybe I just don't see it :/

Reply:
I have added this option when the chatter is on the side part